airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -103,8 +103,37 @@ class RedshiftCreateClusterOperator(BaseOperator):
         "cluster_identifier",
         "cluster_type",
         "node_type",
+        "master_username",
+        "master_user_password",
+        "cluster_type",
+        "db_name",
         "number_of_nodes",
+        "cluster_security_groups",
         "vpc_security_group_ids",
+        "cluster_subnet_group_name",
+        "availability_zone",
+        "preferred_maintenance_window",
+        "cluster_parameter_group_name",
+        "automated_snapshot_retention_period",
+        "manual_snapshot_retention_period",
+        "port",
+        "cluster_version",
+        "allow_version_upgrade",
+        "publicly_accessible",
+        "encrypted",
+        "hsm_client_certificate_identifier",
+        "hsm_configuration_identifier",
+        "elastic_ip",
+        "tags",
+        "kms_key_id",
+        "enhanced_vpc_routing",
+        "additional_info",
+        "iam_roles",
+        "maintenance_track_name",
+        "snapshot_schedule_identifier",
+        "availability_zone_relocation",
+        "aqua_configuration_status",
+        "default_iam_role_arn",
     )
```

Comment:
Not a question, but things like this make me think that we should have these parameters templated by default.

Reply:
Agree